src/swap/SwapScreen.tsx

```diff
@@ -334,7 +334,12 @@ export function SwapScreen({ route }: Props) {
 
   useEffect(() => {
     if (fetchSwapQuoteError) {
-      dispatch(showError(ErrorMessages.FETCH_SWAP_QUOTE_FAILED))
+      Logger.error(TAG, 'Error fetching swap quote', fetchSwapQuoteError.message)
```

Comment:
nit: do we need to send these errors to sentry? we can see them from the gcp logs already. if we want to log them then Logger.info or .warn is good?